lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView.ios.js

```diff
@@ -36,7 +36,6 @@ class KeyboardTrackingView extends PureComponent {
     useSafeArea: PropTypes.bool
   };
 
-  /** V6 should be change to default false */
   static defaultProps = {
     useSafeArea: true
```

Comment:
Why here it's not 'false'?

Reply:
It's a different feature, that we reverted eventually.
(The row I deleted in the spreadsheet)

Reply:
ok